api/src/telemetry/lib/get-report.ts

```diff
@@ -0,0 +1,50 @@
+import { getDatabase } from '../../database/index.js';
+import { useEnv } from '../../env.js';
+import { releaseVersion } from '../../utils/package.js';
+import type { TelemetryReport } from '../types/report.js';
+import { getItemCount } from '../utils/get-item-count.js';
+import { getUserCount } from '../utils/get-user-count.js';
+import { getUserItemCount } from '../utils/get-user-item-count.js';
+
+const basicCountCollections = [
+	'directus_dashboards',
+	'directus_extensions',
+	'directus_files',
+	'directus_flows',
+	'directus_roles',
+	'directus_shares',
+] as const;
+
+/**
+ * Create a telemetry report about the anonymous usage of the current installation
+ */
+export const getReport = async (): Promise<TelemetryReport> => {
+	const db = getDatabase();
+	const env = useEnv();
+
+	const [basicCounts, userCounts, userItemCount] = await Promise.all([
+		getItemCount(db, basicCountCollections),
+		getUserCount(db),
+		getUserItemCount(db),
+	]);
+
+	return {
+		url: env['PUBLIC_URL'],
+		version: releaseVersion,
+		database: env['DB_CLIENT'],
```

Comment:
Thoughts on using `getDatabaseCLient()` instead of `env['DB_CLIENT']` directly, so that `pg` and `postgres` will both be sent as `postgres` rather than as separate values? Not super important as we can always post process this on the telemetry side (something such as `filter rows with 'pg' or 'postgres'` to grab them all), just curious 😄

Reply:
Great suggestion!